src/lib/y2partitioner/format_mount/options.rb

```diff
@@ -0,0 +1,312 @@
+# encoding: utf-8
+
+# Copyright (c) [2017] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "y2storage"
+
+module Y2Storage
+  # FIXME: Monkey patch of Y2Storage::PartitionId it should moved to
+  # yast2-storage-ng
+  class PartitionId
+    include Yast::I18n
+    extend Yast::I18n
+
+    NOT_ALLOW_FORMAT = %i(lvm raid esp prep bios_boot unknown).freeze
```

Comment:
NP: I would prefer the regular syntax, ` [:lvm, :raid, :esp, :prep, :bios_boot, :unknown] `. RuboCop accepts either.

`%i` [follows the upstream style guide.](https://github.com/bbatsov/ruby-style-guide#percent-i) It [is *not* mentioned in the SUSE style guide](https://github.com/SUSE/style-guides/blob/master/Ruby.md#deviations-from-the-upstream-style-guide) but I guess it is an omission because we tried to ban the `%` syntax as too-obscure-for-little-benefit.